src/content/1.7/development/architecture/domain/services.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Domain services
+weight: 20
+---
+
+# Domain services
+
+As mentioned in [CQRS principles](cqrs.md), a single command is just a port, a representation of a single use case, therefore the command handler should not contain any domain logic by itself, but should be orchestrating other services to fulfill the use case instead.
```

Comment:
```suggestion
As mentioned in [CQRS principles]({{< relref "cqrs" >}}), a Command is a representation of a single domain use case. To prevent Command handlers from duplicating code and from depending on each other, they usually don't implement the actual logic themselves, but delegate it to Domain Services.
```